app/models/cuisine.rb

```diff
@@ -1,3 +1,8 @@
 class Cuisine < ApplicationRecord
 	has_many :resturants
+
+	validates_presence_of :title, :image
+	validates_format_of :image, :with => /\.(jpg|png)\Z/i
+	validates_uniqueness_of :title
```

Comment:
Note that you can (and should) add a database index that enforces uniqueness on the title field in the database.

This is complementary to the uniqueness validation you added here.